src/hooks/useWalletSupportsChain/useWalletSupportsChain.ts

```diff
@@ -67,6 +68,7 @@ export const walletSupportsChain: UseWalletSupportsChain = ({
     case gnosisChainId:
       return supportsGnosis(wallet)
     case arbitrumChainId:
+    case arbitrumNovaChainId:
       return supportsArbitrum(wallet)
```

Comment:
should probably add `supportsArbitrumNova` just to be extra cautious?

Reply:
We should, though fine as a follow-up.